source/auth/authentication.txt

```diff
@@ -0,0 +1,250 @@
+
+.. _arch-center-authentication-recs:
+
+========================================
+Guidance for {+service+} Authentication
+========================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta:   
+   :description: Learn about the different authentication mechanisms that Atlas supports.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: onecol
+
+
+|service-fullname| supports a variety of authentication methods to
+ensure robust security. |service| requires all users to authenticate to 
+access the {+atlas-ui+}, |service| databases, and  the {+atlas-admin-api+}. 
+
+.. note:: 
+
+   In this context, a "user" can be a human or an application. We refer to 
+   human users as "Workforce Identity" and applications as 
+   "Workload Identity".
+
+There are two factors in determining which authentication types to use: 
+the identity type (human or machine), and the resource that the identity 
+needs access to (|service| UI, |service| Database, or |service| APIs). 
+
+|service| UI Authentication
+---------------------------
+
+.. note::
+
+   This only applies to Workforce users.
+
+
+Use IP access restrictions, and then either:
+
+- Use :ref:`Federated Authentication <arch-center-federated_auth>` with an SAML 2.0 
+  identity provider, such as Okta, Microsoft Entra ID, or Ping Identity, or 
+- Use Atlas credentials with 
+  :ref:`Multi-factor Authentication (MFA) <arch-center-mfa>`.
+
+|service| Database Authentication
+---------------------------------
+
+Workforce Users
+~~~~~~~~~~~~~~~
+Use :ref:`Workforce Identity Federation <arch-center-workforce_if>`.
```

Comment:
question for @icwill: in this PR it mentions that users must use SCRAM to authenticate to target clusters. I confirmed this in the live migration docs and it appears to be true. Does this deserve a special mention here? https://github.com/mongodb/docs-atlas-architecture/pull/192/files#diff-2110d67577af1481caa9910576e65f01a440257042c4995bd571b26a4d103e2eR39-R40